crates/shared/src/bad_token/token_owner_finder.rs

```diff
@@ -84,6 +90,15 @@ pub struct Arguments {
         default_value = "",
     )]
     pub whitelisted_owners: HashMap<H160, Vec<H160>>,
+
+    /// The solvers urls to query the token owner pairs.
+    #[clap(long, env, use_value_delimiter = true)]
+    pub solver_token_owners_urls: Vec<Url>,
+
+    /// Interval between consecutive queries to update the solver token owner pairs.
+    /// Values should be in pair with `solver_token_owners_urls`
+    #[clap(long, env, use_value_delimiter = true, value_parser = duration_from_seconds)]
+    pub solver_token_owners_cache_update_intervals: Vec<Duration>,
 }
```

Comment:
Not sure if we should force these two values to be paired.

Reply:
We can probably start with defining one value for all `solver_token_owners_urls`, but its fine like this too.